### PR TITLE
chore: polish processing message

### PR DESCRIPTION
**Pull request by lostthing**

### CODE REVIEW

File: gui/src/assets/js/networkInspect.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ function waitingConnected(promise, interval, cancel, timeout) {
             }
           },
           () => {
-            if (res.data.message !== "the last request is being processed") {
+            if (res.data.message !== "Processing last request") {
```
Comment (mzz2017) [inline gui/src/assets/js/networkInspect.js:24]: I think the expression is not intuitive enough and may cause misunderstanding. What do you think?

**Comment by lostthing** [comment]
Yeah but it was a quick chore, to make message more natural when reading. 

Maybe better way is tell user what's processing like "Processing: ping servers", a custom error? 
```go
type ProcessingError struct {
    Type string
}

func (e *ProcessingError) Error() string {
    return fmt.Sprintf("Processing: %v", e.Type)
}
```
This way will change more code, I'm not quite familiar with golang

**Comment by mzz2017** [comment]
@lostthing "Processing" is confusing. I think the core meaning should be fully expressed.

**Comment by mzz2017** [comment]
@lostthing It may be a noun. Or it is on going. Users should guess.

**Pull request closed without merging**